activities.json

```diff
@@ -471,8 +471,8 @@
     "description": "An asynchronous Javascript cookies API for documents and workers",
     "id": "cookie-store",
     "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1475599",
-    "mozPosition": "defer",
-    "mozPositionDetail": "This API provides better access to cookies. However, the improvements also expand access to cookie metadata and the interactions with privacy features like the Storage Access API are unclear. Work on improving cookie interoperability, which is ongoing, might be needed before an assessment can be made.",
+    "mozPosition": "positive",
+    "mozPositionDetail": "This API provides better access to cookies. Due to privacy concerns as outlined in issue 94, we are positive for a subset of this API which only exposes the same information available through document.cookie. At this time we're not ready to expose ServiceWorkerRegistration.cookies or implement the CookieStoreManager as proposed; we will discuss this along with the subset proposal in the appropriate standards group",
```

Comment:
Please outline the privacy concerns inline or remove "Due to privacy concerns as outlined in issue 94,"